extensions/interactions/AlgebraicExpressionInput/protractor.js

```diff
@@ -30,13 +30,13 @@ var expectInteractionDetailsToMatch = async function(elem) {
   ).toBe(true);
   // Testing editor's value in default state.
   expect(
-    await objects.AlgebraicExpressionEditor(elem.element(by.tagName(
+    await objects.MathEditor(elem.element(by.tagName(
       'oppia-interactive-algebraic-expression-input'))).getValue()
   ).toBe('\\color{grey}{\\text{\\small{Type a formula here.}}}');
 };
 
 var submitAnswer = async function(elem, answer) {
-  await objects.AlgebraicExpressionEditor(elem.element(by.tagName(
+  await objects.MathEditor(elem.element(by.tagName(
     'oppia-interactive-algebraic-expression-input'))).setValue(answer);
```

Comment:
ditto: still `algebraic-expression`

Reply:
The MathEditor is common for both interactions, but the tag still needs to be specific to the interaction itself. Am I missing something?